WebDriverAgentLib/Utilities/FBConfiguration.h

```diff
@@ -169,6 +169,24 @@ NS_ASSUME_NONNULL_BEGIN
 + (void)setIncludeNonModalElements:(BOOL)isEnabled;
 + (BOOL)includeNonModalElements;
 
+/**
+ Sets custom class chain locators for accept/dismiss alert buttons location.
+ This might be useful if the default buttons detection algorithm fails to determine alert buttons properly
+ when defaultAlertAction is set.
+
+ @param classChainLocator Valid class chain locator, which determines accept/reject button
+ on the alert. The search root is the alert element itself.
+ Setting this value to nil or an empty string (the default
+ value) will enforce WDA to apply the default algorithm for alert buttons location.
+ If an invalid/non-parseable locator is set then the lookup will fallback to the default algorithm and print a
+ warning into the log.
+ Example: ** /XCUIElementTypeButton[`label CONTAINS[c] 'accept'`]
```

Comment:
no space after `**`?

Reply:
removing a space there will break the multiline comment. Is there any better way to represent the example?

Reply:
ah...I see. I thought `@code/@endcode` worked but did not